src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.cs

```diff
@@ -1609,6 +1611,31 @@ protected override void OnPointerReleased(PointerRoutedEventArgs args)
 			}
 		}
 
+		protected override void OnPointerPressed(PointerRoutedEventArgs pArgs)
+		{
+			// If our templated parent is handling mouse button, we should not take
+			// focus away.  They're handling it, not us.
+			if (m_templatedParentHandlesMouseButton)
+			{
+				return;
+			}
+
+			var spPointerPoint = pArgs.GetCurrentPoint(this);
+			if (spPointerPoint is null)
+			{
+				return;
+			}
+			var spPointerProperties = spPointerPoint.Properties;
+			if (spPointerProperties is null)
+			{
+				return;
+			}
+			m_isPointerLeftButtonPressed = spPointerProperties.IsLeftButtonPressed;
+
+
+			// Don't handle PointerPressed event to raise up
+		}
```

Comment:
Should this be in the `MuxInternal` file ? (BTW: with uno `spPointerPoint` and `spPointerProperties` cannot be `null`)

Reply:
Yeah, I thought it would be nice to keep the same structure from WinUI, but it's unnecessary noise indeed.